src/main/fc/core.c

```diff
@@ -277,7 +277,7 @@ void updateArmingStatus(void)
             // We also need to prevent arming until it's possible to send DSHOT commands.
             // Otherwise if the initial arming is in crash-flip the motor direction commands
             // might not be sent.
-            && dshotCommandsAreEnabled(DSHOT_CMD_TYPE_INLINE)
+            && (dshotCommandsAreEnabled(DSHOT_CMD_TYPE_INLINE) || motorConfig()->dev.motorPwmProtocol == PWM_TYPE_DISABLED)
```

Comment:
I'd probably even go for `(!isMotorProtocolDshot() || dshotCommandsAreEnabled(DSHOT_CMD_TYPE_INLINE))` - no point to wait for Dshot commands for any non-Dshot protocols.

Reply:
Makes sense. Updated.